iTechArt.Repository/PaginationHelpers/Pagination.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace iTechArt.Repository.PaginationHelpers
+{
+    internal static class Pagination
+    {
+        /// <summary>
+        /// Paginates DbSet according to the pageIndex and PageSize
+        /// </summary>
+        public static IQueryable<TSource> Paginate<TSource>(this IQueryable<TSource> source, int pageIndex)
+        {
+            return PaginationConstant.PageSize > 0 && pageIndex > 0
+                ? source.Skip((pageIndex - 1) * PaginationConstant.PageSize).Take(PaginationConstant.PageSize)
```

Comment:
PaginationConstant.PageSize it is constant, why you compare it with 0?

Reply:
done